iterations_runners/iterations_runner.lua

```diff
@@ -164,7 +163,11 @@ for BM_i = 1, BM_iters, 1 do
     end
 end
 
-krun_done()
+-- In LuaJIT, FFI functions are cdata values that are unable to reference any other object owned by
```

Comment:
One last question: In the Lua world, wha tis the accepted line length?

Reply:
idk i just based it the line lengths you already had in the file that long and just went with that

Reply:
OK, I might do a formatting pass on this file at some point.